lib/Doctrine/DBAL/Driver/PDOOracle/Driver.php

```diff
@@ -10,8 +10,7 @@
 /**
  * PDO Oracle driver.
  *
- * WARNING: This driver gives us segfaults in our testsuites on CLOB and other
- * stuff. PDO Oracle is not maintained by Oracle or anyone in the PHP community,
+ * WARNING: PDO Oracle is not maintained by Oracle or anyone in the PHP community,
```

Comment:
As well as the previous version of this warning, it may become invalid over time. I'd rather let the end-user decide which of the drivers they want to use and remove this warning entirely.

Reply:
removed